### DISCUSSION TITLE
About customizing scraper settings for location-based SERP scraping

### DISCUSSION THREAD

**Discussion by nu-ad** [towfiqi/serpbear#151] [Ideas]
Hi!

I am reaching out to ask about customizing the scraper settings within the SERPBEAR framework, specifically for the scrapers with API's that support the adjustements I require.

My goal is to enhance the scraper's capability to consider advanced location parameters more effectively during SERP scraping, such as defining a region/area or a city.

I have reviewed the current configurations of the scrapers settings, which currently includes parameters such as country and lang within the scrapeURL function. This setup seems to primarily focus on country-level scraping.

I am considering adjustments that would enable more precise location-based scraping, potentially down to the city or region level, according to proper API docs. Would integrating a more granular location parameter into the scrapeURL function be sufficient for this purpose? If so, could you share your thoughts  on the best approach to implement this? Perhaps this is a feature on the road-map of SEPRBEAR?

Additionally, I would appreciate any insights or suggestions you might have on optimizing the scraper for location-specific SERP data, ensuring accurate and efficient data collection.

Thank you for your time and assistance.

Kind regards,
J

**Comment by towfiqi** [comment]
Yes, you are correct to assume it's in the roadmap. Probably will release this feature next month.

**Comment by towfiqi** [comment]
This has been added in v1.0.0 which has just been released. You can now scrape city-level Google search results with the following scrapers:

searchapi.io
serpapi.com
serper.dev
spaceserp.com
valueserp.com

**Reply by nu-ad** [reply]
This is great news! Well done! Off to checking this out!

**Reply by jchambo** [reply]
@towfiqi Is this list up to date? Maybe add / update the SerpBear website with this information? I had to hunt around for an answer to this as I've been using Scraping Robot as my API source, but it sounds like I'd have to switch my integration in order to leverage Local SEO results?
